clang-tools-extra/test/clang-query/Inputs/runtime_file.script

```diff
@@ -0,0 +1 @@
+m functionDecl()
```

Comment:
Can you add a few more commands to the file to show that we execute all of the commands rather than just one?

It would also be good to have a test where the file is not found and a test where the file is empty, just to test boundary conditions.